lib/mpl_toolkits/axisartist/grid_helper_curvelinear.py

```diff
@@ -140,7 +138,7 @@ def update_lim(self, axes):
 
         grid_finder = self.grid_helper.grid_finder
 
-        #e1, e2 = self._extremes # ranges of other coordinates
+        # e1, e2 = self._extremes # ranges of other coordinates
```

Comment:
goes away?

Reply:
duplicated just a bit before in the same function so yes